app/renderer/components/config_menu.js

```diff
@@ -49,8 +50,25 @@ module.exports = {
         </div>
     </div>
     `,
-
-    methods: {
+   methods: {
+        importTasks() {
+            dialog.showOpenDialog({
+                filters: [{
+                    name: 'json',
+                    extensions: ['json']
+                }]
+            }, (fileNames) =>{
+                if (!filename || !FileNames[0]) return;
```

Comment:
empty return is not very nice, maybe the opposite condition and wrapping everything else inside the if?
```
if(filenames && fileNames[0]){
...

```

Also, a couple of things:
1. Use a beautifier or something to avoid incoherent tabs, this if statement is not properly aligned
2. Should the first element be `fileNames`? I mean, filename is not yet defined 
3. filename is just one word, it should not be formatted as fileName